Useful Scripts/PowerPack backup to Git/PowerPack backup to Git.ps1

```diff
@@ -0,0 +1,88 @@
+Import-Module SqlServer
+<#
+    .NOTES
+    ===========================================================================
+    Created with:	Visual Studio Code
+    Created on:		28-03-2023
+    Created by:		MARA
+    Organization:	IT-Center Fyn
+    Filename:		PowerPack backup to Git.ps1
+    ===========================================================================
+    .DESCRIPTION
+        TODO: A description of the file.
+#>
+##################
+### PARAMETERS ###
+##################
+# DO NOT CHANGE
+# Change as needed
+$DevCMPID = '2'
+$SqlServer = 'CISRVKURSUS'
```

Comment:
Erm....skal dette ikke være vores rigtige server, i stedet for din kursus server?